x-pack/plugins/secops/public/components/timeline/footer/index.tsx

```diff
@@ -151,15 +152,29 @@ export class Footer extends React.PureComponent<FooterProps, FooterState> {
   public readonly state = {
     isPopoverOpen: false,
     paginationLoading: false,
+    updatedAt: null,
   };
 
   public componentDidUpdate(prevProps: FooterProps) {
-    const { paginationLoading } = this.state;
-    const { isLoading } = this.props;
+    const { paginationLoading, updatedAt } = this.state;
+    const { isLoading, getUpdatedAt } = this.props;
     if (paginationLoading && prevProps.isLoading && !isLoading) {
       this.setState({
         ...this.state,
         paginationLoading: false,
+        updatedAt: getUpdatedAt(),
+      });
+    }
+    if (updatedAt === null) {
+      this.setState({
+        ...this.state,
+        updatedAt: getUpdatedAt(),
+      });
```

Comment:
Did you intend for both of these conditions to have `getUpdatedAt()` or was this going to be `updatedAt`?

Right now they both just call `getUpdatedAt()`. Pointing it out in case that's not what you wanted.

Reply:
All good there!!!